### PR TITLE
fixed timelines tests for infra provider for 5.6.2.2

### PR DESCRIPTION
**Pull request by izapolsk**
{{ pytest: cfme/tests/infrastructure/test_timelines.py }}

### CODE REVIEW

**Comment by dajoRH** [comment]
Lint report for commit 54998d4c5b9c11e629094f8521b53894e9950e49:

Everything seems all right from lint perspective :smile: .

Commit inspection report:

**No commit flaws detected**.
<mergeable/>

*CFME QE Bot*

File: cfme/common/vm.py [hunk 1]
```diff
@@ -145,9 +145,9 @@ def factory(cls, vm_name, provider, template_name=None, template=False):
     TO_OPEN_EDIT = None  # Name of the item in Configuration that puts you in the form
     QUADICON_TYPE = "vm"
     # Titles of the delete buttons in configuration
-    REMOVE_SELECTED = {version.LOWEST: 'Remove selected items from the VMDB',
+    REMOVE_SELECTED = {'5.6.2.2': 'Remove selected items from the VMDB',
```
Comment (mshriver) [inline cfme/common/vm.py]: I think that's in alignment with the read-the-docs style guide, to move first item in the dictionary to a newline so the keys line up.

**Comment by dajoRH** [comment]
Lint report for commit b49eb25ce9fece1632410acf3a1dc71d2413b6d8:

Everything seems all right from lint perspective :smile: .

Commit inspection report:

**No commit flaws detected**.
<mergeable/>

*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit b950f33181ed07cd12db9dd0f5b7a66045affd9e:

Everything seems all right from lint perspective :smile: .

Commit inspection report:

**No commit flaws detected**.
<mergeable/>

*CFME QE Bot*

**Comment by psav** [comment]
Can we put PRT meta data in there so it tests something useful?

**Comment by dajoRH** [comment]
Lint report for commit 7f1738d2374542e2bd36b5c877484e151c25824c:

Everything seems all right from lint perspective :smile: .

Commit inspection report:

**No commit flaws detected**.
<mergeable/>

*CFME QE Bot*

**Pull request merged**